src/screens/ChatScreen/ChatScreen.js

```diff
@@ -0,0 +1,327 @@
+      return;
+    }
+    const uri = setImage;
+    const childPath = `forum/${this.state.data}/${
+      firebase.auth().currentUser.uid
+    }/${Math.random().toString(36)}`;
+    const response = await fetch(uri);
+    const blob = await response.blob();
+
+    const task = firebase.storage().ref().child(childPath).put(blob);
+
+    const taskCompleted = () => {
+      const {transferred, setDisplayImage} = this.state;
+      task.snapshot.ref
+        .getDownloadURL()
+        .then(snapshot => {
+          this.setState({setDisplayImage: snapshot});
+        })
+        .catch(error => {
+          alert(error);
+        });
+    };
+
+    const taskError = snapshot => {
+      alert("Error Occured");
+    };
+
+    task.on(
+      "state_changed",
+      snapshot => {
+        const {transferred} = this.state;
+        this.setState({
+          transferred:
+            Math.round(snapshot.bytesTransferred / snapshot.totalBytes) * 10000,
+        });
+      },
+      taskError,
+      taskCompleted,
+    );
+  };
+
+  writeUserData(message) {
+    const {profilePresent, setDisplayImage} = this.state;
+    var timeDate = moment();
+    firebase
+      .firestore()
+      .collection("users")
+      .doc(firebase.auth().currentUser.uid)
+      .get()
+      .then(doc => {
+        firebase
+          .database()
+          .ref(this.state.data + "/")
+          .push({
+            name: doc.data().fullName,
+            message,
+            creation: new Date().toUTCString(),
+            messageTime: timeDate.format("lll"),
+            imageURL: profilePresent ? doc.data().downloadURL : NO_IMAGE_LINK,
+            messageImage: setDisplayImage,
+          })
+          .then(data => {
+            this.setState({
+              message: "",
+              messageImage: null,
+              setImage: null,
+              setDisplayImage: null,
+            });
+          })
+          .catch(error => {
+            alert(error);
+          });
+      });
+  }
+
+  renderData() {
+    const {messageList} = this.state;
+    firebase
+      .database()
+      .ref(this.state.data + "/")
+      .on("value", snapshot => {
+        var messageArray = [];
+        this.setState({messageList: Object.values(snapshot.val())});
+      });
+  }
+
+  showAlert = () => {
+    const {setImage, setDisplayImage} = this.state;
+    Alert.alert("Alert", "", [
+      {text: "Show", onPress: () => Linking.openURL(setDisplayImage)},
+      {
+        text: "Remove",
+        onPress: () => {
+          this.setState({
+            setDisplayImage: null,
+            setImage: null,
+          });
+        },
+      },
+      {text: "Cancel", onPress: () => null, style: "cancel"},
+    ]);
+  };
+
+  render() {
+    const {
+      messageList,
+      profilePresent,
+      setImage,
+      setDisplayImage,
+      messageImage,
+      message,
+    } = this.state;
+    return (
+      <>
+        <SafeAreaView style={{flex: 0, backgroundColor: "#51AD28"}} />
```

Comment:
It would be great of you can move all the css styles into a single styles file for this view.